addons/binding/org.openhab.binding.connectsdk/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="connectsdk" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+	<name>ConnectSDK Binding</name>
```

Comment:
This was the intend. However the project is abandoned since 2015 and LG released all of the development team. Last version 1.6.0.  

I personally tried to get Chromecast and AppleTV to work, without success. I suspect Chromecast and AppleTV interfaces/protocols have changed since 1.6.0.

I forked the lib and created a pure java version of it, removing all the Android dependencies. It works with all newer WebOS smart TVs. 

There is already an "LG binding" for older Netcast 3.0 and Netcast 4.0 models.

Reply:
Thanks for explanation. I think it would be nice to have LG somewhere in the name (`<name></name>`) of the binding. I can imagine an owner of a LG TV will look for a binding by typing LG in the addons search box. This should make it easier to find it and use it. The package name is not a problem.

Reply:
Done